test/compress/hoist_props.js

```diff
@@ -827,3 +827,25 @@ issue_2519: {
     }
     expect_stdout: "5.5"
 }
+
+toplevel_const: {
+    options = {
+        hoist_props: true,
+        reduce_vars: true,
```

Comment:
Could you please add a redundant `toplevel: false,` here for emphasis?

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/2724/commits/0e6ac6d29348b4ab3f992c7661a55f7b3d59f4e4